### ISSUE TITLE
Docs in PyPI tarball are broken

### ISSUE THREAD

**Issue by QuLogic** [blink1073/oct2py#117]
The PyPI tarball of 4.0.2 contains a weird set of files for the documentation. I believe all the files in `docs/source` are there, but the `docs/_templates` directory is missing (not sure if it's needed). The two `makefile`/`make.bat` are missing. Then there's `docs/_build/html`, but it contains _only_ `*.png` files.

**Comment by QuLogic** [comment]
PS, you _can_ build the docs using `sphinx-build` directly; it's just weird to have half-built `_build` directory.

Also, there's this warning:
```
/usr/lib/python3.5/site-packages/numpydoc/docscrape.py:120: UserWarning: Unknown section Notest
```
which appears to be coming from [here](https://github.com/blink1073/oct2py/blob/e485da051b48cb5d49220daeb546b359a2d8a559/oct2py/core.py#L222).

**Comment by blink1073** [comment]
Thanks, fixed in f5060bb445d2b974b6545945424a433910807e1a, released in https://github.com/blink1073/oct2py/releases/tag/v4.0.3.
